README.md

```diff
@@ -4,4 +4,4 @@ The Checkout.com 3D Secure (3DS) mobile SDK allows you to provide a native 3DS2
 <br/>
 The SDK handles the device data collection, communication with the card issuer, and presentation of 3D Secure challenges to the customer when required.
 <br/><br/>
-👉  [See the Integration Guide with Code Examples](https://docs.checkout.com/3d-secure-mobile-sdks)
\ No newline at end of file
+👉  [See the Integration Guide with Code Examples](https://checkout.com/docs/risk-management/3d-secure/sessions/non-hosted-sessions/3d-secure-mobile-sdks)
```

Comment:
```suggestion
👉  [See the Integration Guide with Code Examples](https://www.checkout.com/docs/risk-management/3d-secure/sessions/non-hosted-sessions/3d-secure-mobile-sdks#Integrate_the_3D_Secure_SDK_into_your_app)
```

The page is quite long, this suggestion link will get consumer straight to the "integration guide" section

Reply:
The doc seems to suggest Cocoapods integration. Can you create a task in the backlog for us to add SPM integration?

@deepesh-vasthimal-cko would be nice to get any new users possibility to not get tied into Cocoapods as soon as possible :)